packages/react-core/src/components/Toolbar/ToolbarToggleGroup.tsx

```diff
@@ -7,32 +7,73 @@ import { ToolbarContext, ToolbarContentContext } from './ToolbarUtils';
 import { Button } from '../Button';
 import globalBreakpointLg from '@patternfly/react-tokens/dist/js/global_breakpoint_lg';
 
-import { ToolbarBreakpointMod } from './ToolbarUtils';
-import { formatBreakpointMods, capitalize, toCamel } from '../../helpers/util';
-import { PickOptional } from '../../helpers/typeUtils';
+import { formatBreakpointMods, toCamel } from '../../helpers/util';
 
 export interface ToolbarToggleGroupProps extends ToolbarGroupProps {
   /** An icon to be rendered when the toggle group has collapsed down */
   toggleIcon: React.ReactNode;
-  /** The breakpoint at which the toggle group is collapsed down */
-  breakpoint: 'md' | 'lg' | 'xl';
-  /** An array of objects representing the various modifiers to apply to the data toolbar toggle group at various breakpoints */
-  breakpointMods?: ToolbarBreakpointMod[];
+  /** Visibility at various breakpoints. */
+  visiblity?: {
+    default?: 'hidden' | 'visible';
+    md?: 'hidden' | 'visible';
+    lg?: 'hidden' | 'visible';
+    xl?: 'hidden' | 'visible';
+    '2xl'?: 'hidden' | 'visible';
+  };
+  /** Controls when filters are shown and when the toggle button is hidden. */
+  show?: {
+    default?: 'show';
```

Comment:
this only has show?

Reply:
Yep, it replaces the `breakpoint` prop! Similarly Flex has single `grow` and `shrink` props. I think it's fine since in most cases they're strings and to keep it consistent.